app/build.gradle.kts

```diff
@@ -155,7 +155,11 @@ dependencies {
     // features
     implementation(project(":features:cells"))
     implementation(project(":features:sketch"))
+    implementation(projects.features.sync)
```

Comment:
We need to add this to the kover section 👀 

 i.e.
`kover(project((projects.features.sync))`